imqsauth/http.go

```diff
@@ -121,6 +122,24 @@ type ImqsCentral struct {
 	subscriberLock sync.RWMutex
 }
 
+func (x *ImqsCentral) IsLockable(identity string) (bool, error) {
```

Comment:
We should probably add a comment above this function explaining why we don't allow admin accounts to be locked out. And also mention that our inter-service accounts are also admins, and it would be terrible to allow "machine" accounts to be lockable.